.travis.yml

```diff
@@ -54,3 +56,13 @@ jobs:
         on:
           tags: true
           go: 1.11.x
+    - name: 'Deploy to staging'
+      stage: staging
+      script:
+        - make packages
+        - make docker-push
```

Comment:
apparently, it's not going to work because we have a check for master branch in ci makefile.

```
	@if [ "$(BRANCH)" == "master" ]; then \
		echo "docker-push is disabled on master branch" \
		exit 1; \
	fi; \
```

I absolutely forgot about it.

Reply:
https://github.com/src-d/ci/pull/94